test/rtest-gobject-signals.lisp

```diff
@@ -3,6 +3,10 @@
 (def-suite gobject-signals :in gobject-suite)
 (in-suite gobject-signals)
 
+;; Ensure that GtkButton type is registered, otherwise there will be test
+;; failures at the first run
+(make-instance 'gtk-button)
```

Comment:
Ah, no, we should do this differently, via `(glib::at-init () (foreign-funcall "xyz..._get_type" :int))` or even create a new pattern for that so it doesn't get out of hand.

Reply:
I tried your solution, and it does not seem to work. I still get test failures, with several messages like the following on the console:

`(Program:20846): GLib-GObject-WARNING **: /build/glib2.0-B1uXKV/glib2.0-2.50.3/./gobject/gsignal.c:1253: unable to lookup signal "activate" of unloaded type 'GtkButton'`

Reply:
I have trouble replicating this, could you post what you're running to get there?

Reply:
I just run the whole testsuite (or just the tests in the `gobject-signals` sub-testsuite).

The important point is to start from a freshly restart Lisp instance, where no `GtkButton` has ever been instantiated. And I use SBCL.